data/presets/power/terminal.json

```diff
@@ -0,0 +1,28 @@
+{
+    "icon": "temaki-power_device",
+    "fields": [
+        "operator",
+        "line_attachment"
+    ],
+    "moreFields": [
+        "height",
+        "ref"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "electric terminal",
+        "electrical terminal",
+        "service entrance"
+    ],
+    "tags": {
+        "power": "terminal"
+    },
+    "addTags": {
+        "power": "terminal",
+        "line_attachment": "anchor"
```

Comment:
Are you sure that `line_attachment=anchor` should be added to every single `power=terminal`?

If mass adding such tag is useful, it should be done as an automated bot edit. Though if it is really supposed to appear at every single one then I am dubious is there point in doing it.

Not via deprecation rule that wastes enormous amount of human time for no good reason.

Yes, I know that we have some rules like that in this repo already. I think that they are a horrible mistake and should be removed.

Reply:
I'm not really sure I understand your comment :thinking: 

It's not a depreciation rule, it's an additional tag that is relevant (and recommended) to add to this object, and it seems to me that this corresponds perfectly to what addTags is for. 
<img width="910" height="90" alt="image" src="https://github.com/user-attachments/assets/a3dad562-b2ee-4301-b170-4b1c97d8c1ac" />

I don't understand why you suggest not adding it.

Reply:
it it needs to be mass added to 64k objects (see https://overpass-turbo.eu/s/2hpJ ) then it should be done with a proper bot edit, rather than undiscussed distributed bot edit with people adding it via "upgrade" button in iD

(also, if it absolutely always applies then I do not get what is the point of adding it to every object)